### ISSUE TITLE
RXPTest - Components - Image - APIs - iOS reports wrong image sizes

### ISSUE THREAD

**Issue by sbeca** [Microsoft/reactxp#1047]
Testing the iOS version of RXPTest on:
- macOS 10.14.3
- Xcode 10.1
- Building for iPhone X simulator
- Building using command `cd ./samples/RXPTest/ && npm i && npm run ios`

I get the following error:
![Screenshot1](https://user-images.githubusercontent.com/1635228/54069914-3fe20d80-42af-11e9-8277-41994c085858.png)

Using the expanded error logging I added in PR #1046, I get the following errors:
![Screenshot2](https://user-images.githubusercontent.com/1635228/54069926-5d16dc00-42af-11e9-91bd-bc4fd31258bb.png)

So the difference between received dimensions and expected dimensions is a scale factor of 3x. This lines up with the screen scale factor of the iPhone X. However, this seems like a weird way to report image resolution so should be investigated further.

**Comment by sbeca** [comment]
@erictraut, I don't think this issue was meant to be closed yet. Sorry if I wasn't clear in my initial messaging of this issue but PR #1046 doesn't fix this issue, it just makes it easier to see what is going wrong in the error logs. Unless the different way that image dimensions are reported on iOS is correct and the errors in RXPTest shouldn't be shown in the first place?

**Comment by fbartho** [comment]
Is this still a concern @sbeca?
